src/index.ts

```diff
@@ -1,5 +1,6 @@
 import { readFileSync } from "fs";
 import * as core from "@actions/core";
+import * as github from "@actions/github";
```

Comment:
The import statement for `@actions/github` should be reviewed for necessity. If it's not used in the code that follows, it should be removed to keep the code clean.